prime-router/src/main/java/fhirengine/translation/hl7/structures/radxmars251/segment/OBX.java

```diff
@@ -0,0 +1,1463 @@
+public class OBX extends AbstractSegment {
+
+  /**
+   * Creates a new OBX segment
+   */
+  public OBX(Group parent, ModelClassFactory factory) {
+    super(parent, factory);
+    init(factory);
+  }
+
+  private void init(ModelClassFactory factory) {
+    try {
+      this.add(SI.class, false, 1, 4, new Object[]{getMessage()}, "Set ID - OBX");
+      this.add(ID.class, false, 1, 3, new Object[]{getMessage(), Integer.valueOf(125)}, "Value Type");
+      this.add(CWE.class, true, 1, 0, new Object[]{getMessage()}, "Observation Identifier");
+      this.add(ST.class, false, 1, 0, new Object[]{getMessage()}, "Observation Sub-ID");
+      this.add(Varies.class, false, 0, 0, new Object[]{getMessage()}, "Observation Value");
+      this.add(CWE.class, false, 1, 0, new Object[]{getMessage()}, "Units");
+      this.add(ST.class, false, 1, 0, new Object[]{getMessage()}, "References Range");
+      this.add(CWE.class, false, 0, 0, new Object[]{getMessage()}, "Interpretation Codes");
+      this.add(NM.class, false, 1, 0, new Object[]{getMessage()}, "Probability");
+      this.add(ID.class, false, 0, 2, new Object[]{getMessage(), Integer.valueOf(80)}, "Nature of Abnormal Test");
+      this.add(ID.class, true, 1, 1, new Object[]{getMessage(), Integer.valueOf(85)}, "Observation Result Status");
+      this.add(DTM.class, false, 1, 0, new Object[]{getMessage()}, "Effective Date of Reference Range");
+      this.add(ST.class, false, 1, 0, new Object[]{getMessage()}, "User Defined Access Checks");
+      this.add(DTM.class, false, 1, 0, new Object[]{getMessage()}, "Date/Time of the Observation");
+      this.add(CWE.class, false, 1, 0, new Object[]{getMessage()}, "Producer's ID");
+      this.add(XCN.class, false, 0, 0, new Object[]{getMessage()}, "Responsible Observer");
+      this.add(CWE.class, false, 0, 0, new Object[]{getMessage()}, "Observation Method");
+      this.add(EI.class, false, 0, 0, new Object[]{getMessage()}, "Equipment Instance Identifier");
+      this.add(DTM.class, false, 1, 0, new Object[]{getMessage()}, "Date/Time of the Analysis");
+      this.add(CWE.class, false, 0, 0, new Object[]{getMessage()}, "Observation Site");
+      this.add(EI.class, false, 1, 0, new Object[]{getMessage()}, "Observation Instance Identifier");
+      this.add(CNE.class, false, 1, 0, new Object[]{getMessage()}, "Mood Code");
+      this.add(XON.class, false, 1, 0, new Object[]{getMessage()}, "Performing Organization Name");
+      this.add(XAD.class, false, 1, 0, new Object[]{getMessage()}, "Performing Organization Address");
+      this.add(XCN.class, false, 1, 0, new Object[]{getMessage()}, "Performing Organization Medical Director");
+      this.add(ID.class, false, 1, 10, new Object[]{getMessage(), Integer.valueOf(909)}, "Patient Results Release Category");
+      this.add(CWE.class, false, 1, 0, new Object[]{getMessage()}, "Root Cause");
+      this.add(CWE.class, false, 0, 5, new Object[]{getMessage()}, "Local Process Control");
```

Comment:
Choosing to defer to the HL7 v2.8 spec in classifying OBX-28 as CWE. RADxMARS classes this as ID, but as an unused optional field and it can be processed as CWE with no issue.